test/test_graphs.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+from graphviz2drawio import graphviz2drawio
+import sh
+
+@pytest.mark.parametrize('file', [
+    './directed/hello.gv.txt',
+])
+def test_gv_file(file):
+    xml = graphviz2drawio.convert(file)
+    print(xml)
```

Comment:
this should assert what the XML contents ought to be instead of printing

Reply:
Well that's a bit tricky. Not sure what to look for in the output to check if it makes sense or not.

If there were more fine grained unit tests to check the svg representation of small patterns against output from dot that would make sense, but maybe as a separate and more long term effort.

I was just thinking that you could catch exceptions and segfaults (I'm seeing a lot of segfaults) and manually eyeball for aesthetics as a first step.

Reply:
@jcaplan once we visually verified the XML output in draw.io, we would then create an xml file with that output and compare the file contents. Could start with your egg graph.  And yes to more fine-grained unit tests! Definitely on the todo list. But I suppose this is an ok start.

Reply:
This is a bit more of a sanity check. Except it doesn't always pass because the order of the nodes is not currently deterministic. The nodes need to be consistently ordered in the xml to make this sort of thing stable.